signature/policy_eval_signedby_test.go

```diff
@@ -130,26 +137,30 @@ func TestPRSignedByIsSignatureAuthorAccepted(t *testing.T) {
 	require.NoError(t, err)
 	sig, err := ioutil.ReadFile("fixtures/unknown-key.signature")
 	require.NoError(t, err)
-	// Pass a nil pointer to, kind of, test that the return value does not depend on the image parmater..
-	sar, parsedSig, err = pr.isSignatureAuthorAccepted(nil, sig)
-	assertSARRejected(t, sar, parsedSig, err)
+	// Pass a nil pointer to, kind of, test that the return value does not depend on the
+	// image parameter..
+	sar, parsedSig, err = pr.isSignatureAuthorAccepted(m, nil, sig)
+	// FIXME: The error is: errors.InvalidArgumentError("no armored data found") but for
```

Comment:
@mtrmac I will need some help here :-) I think it still works fine, but for some reason the error is not what you are expecting.